object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,23 @@
+/* exported defaults */
+function defaults(target, source) {
+  var tempCount = 0;
+  for (var i in source) {
+    for (var j in target) {
+      if (i === j) {
+        tempCount++;
+      }
+    }
+    if (tempCount === 0) {
+      target[i] = source[i];
+    }
+    tempCount = 0;
```

Comment:
You can likely remove this variable. In the conditional I mentioned in my comment above, you can just do this work if there is no value currently in `target` for the current property in the `for...in` loop